### DISCUSSION TITLE
Separate User and Admin tables.

### DISCUSSION THREAD

**Discussion by mpixelz** [Laravel-Backpack/community-forum#1177] [Q&A (Help)] [answered]
Hi,
Im starting a project on the latest builds of laravel and backpack. 
By default, we only get USERS table with authentication.. BUT i would like to separate them into admins and users. There is some information in the docs which touch base on the topic but i cant get that to work with the newer builds.

Is there any article or instructions on how to separate them for laravel 11 and backpack 6?

any help would be really appreciated.
Thank you.

**Comment by mpixelz** [comment]
So, for now what ive done is.. fresh installed laravel.. using breeze for authentication..
replicated User model and migration to Admin.
Installed backpack and configured it to use admin model instead of user table.
so far seems to work.. 

if anyone can suggest something better or if im doing it wrong please do advice.
Thanks.

**Comment by siberfx** [comment]
you mean you want to use separate models or you want to use and breeze for frontend logins, backpack for admin ? maybe some more information would be helpful to understand the issue

**Reply by mpixelz** [reply]
I want to have my frontend users and backend users separate. so users table refer to frontend users and admin table refers to admin team for the backend operations.

**Reply by siberfx** [reply, marked as answer]
so here you can define your model hardcoded or through your auth.php file as in the image below for a different table as;
![image](https://github.com/user-attachments/assets/a3d8ddb2-f57e-4b93-a751-9cfa220ffbea)
and also you can separate your defined guard in the same file as
![image](https://github.com/user-attachments/assets/474ea24b-22f5-4ed7-9e10-cc4e15bd721d)

these are the ways how you can manage multiple auth guards and separate tables for admin, and frontend users

**Reply by mpixelz** [reply]
yeah as i mentioned in my prev comment.. the method working so far. 
thank you :)
